src/commons/utils/Hooks.ts

```diff
@@ -119,7 +119,18 @@ export const useDimensions = (ref: RefObject<HTMLElement>): [width: number, heig
  */
 export const useResponsive = () => {
   const isMobileBreakpoint = useMediaQuery(`(max-width: ${Constants.mobileBreakpoint}px)`);
+
+  // Based on values from flexboxgrid-helpers
```

Comment:
With the new breakpoints added for the various sizes, can we consider replacing the `isMobileBreakpoint` and `isDesktopBreakpoint` to use these breakpoints instead?

Reply:
Maybe not now, seems out of scope for this PR. But definitely we should try to simplify all these and combine stuff together.